src/r_utils.h

```diff
@@ -9,6 +9,14 @@
 #include <string>
 #include <vector>
 
+#ifndef R_PRIdXLEN_T
+#  ifdef LONG_VECTOR_SUPPORT
+#    define R_PRIdXLEN_T "td"
+#  else
+#    define R_PRIdXLEN_T "d"
+#  endif
+#endif
+
```

Comment:
Inspired by https://github.com/wch/r-source/blob/75cd3117e23f2010634200875f2ef50a83461b8f/src/include/Rinternals.h#L71C1-L80

Who/what is going to define `LONG_VECTOR_SUPPORT` 🤔 ?

Reply:
I suppose it comes via `cpp11/R.hpp`, which includes `Rinternals.h`.

Reply:
Yeah, it is coming from the R headers, you are allowed to use it.